jekyll/_cci2/testing-ios.md

```diff
@@ -297,39 +186,112 @@ workflows:
 
-The following test tools are known to work well on CircleCI
-(though many others should work just fine):
+### Images using macOS 10.15 (Xcode 11.2) and later
 
-* [XCTest](https://developer.apple.com/library/ios/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/01-introduction.html)
-* [Kiwi](https://github.com/kiwi-bdd/Kiwi)
-* [KIF](https://github.com/kif-framework/KIF)
+The [`chruby`](https://github.com/postmodern/chruby) program is installed on the image and can be used to select a version of Ruby. The auto-switching feature is not enabled by default. To select a version of Ruby to use, add the `chruby` function to `~/.bash_profile`:
+
+```yaml
+# ...
+run:
+  name: Set Ruby Version
+  command: echo 'chruby ruby-2.6' >> ~/.bash_profile
+```
 
-### Other Tools
+Replace `2.6` with the version of Ruby required - you do not need to specify the full Ruby version, `2.6.5` for example, just the major version. This will ensure your config does not break when switching to newer images that might have slightly newer Ruby versions.
+
+### Images using macOS 10.14 (Xcode 11.1) and earlier
+
+Images using macOS 10.14 and earlier (Xcode 11.1 and earlier) have both `chruby` and [the auto-switcher](https://github.com/postmodern/chruby#auto-switching) enabled by default.
+
+To specify a version of Ruby to use, there are two options. You can [create a file named `.ruby-version` and commit it to your repository, as documented by `chruby`](https://github.com/postmodern/chruby#auto-switching). 
+
+If you do not want to commit a `.ruby-version` file to source control, then you can create the file from a job step:
+
+```yaml
+# ...
+run:
+  name: Set Ruby Version
+  command:  echo "ruby-2.4" > ~/.ruby-version
+```
+
+Replace `2.4` with the version of Ruby required - you do not need to specify the full Ruby version, `2.4.9` for example, just the major version. This will ensure your config does not break when switching to newer images that might have slightly newer Ruby versions.
+
+### Installing additional Ruby versions
+
+To run a job with a version of Ruby that is not pre-installed, you must install the required version of Ruby. We use the [ruby-install](https://github.com/postmodern/ruby-install) tool to install the required version. After the install is complete, you can select it using the appropriate technique above.
+
+### Using Custom Versions of CocoaPods and Other Ruby Gems
 {:.no_toc}
 
-Popular iOS testing tools like [Appium](http://appium.io/) and [Frank](http://www.testingwithfrank.com/) should also
-work normally and are installed and called using `run` commands.
+To make sure the version of CocoaPods that you use locally is also used
+in your CircleCI builds, we suggest creating a Gemfile in your iOS
+project and adding the CocoaPods version to it:
+
+```ruby
+source 'https://rubygems.org'
+
+gem 'cocoapods', '= 1.3.0'
+```
+
+Then you can install these using bundler:
+
+```yaml
```

Comment:
this needs to be surrounded with 
`{% raw %}`
 > ... code
`{% endraw %}`

because there are unescaped `{{` and `}}` in the code block (Jekyll is trying to evaluate it).